### PR TITLE
make radio button container clickable

### PR DESCRIPTION
**Pull request by liampuk**
## What does this do?

Makes the container of the radio button clickable, instead of just the radio button element.

### CODE REVIEW

File: src/ConfirmationRadioButtons/RadioButtonStyled.tsx [hunk 1]
```diff
@@ -28,7 +27,7 @@ export const RadioButton: FC<RadioButtonProps> = ({
   onBlur,
   disabled = false,
 }: RadioButtonProps) => (
-  <Box flex alignItems="center">
+  <InputContainer>
```
Comment (fleonard) [inline src/ConfirmationRadioButtons/RadioButtonStyled.tsx:30]: instead of adding a new StyledComponent you can just use `as="label"`

**Review by fleonard — COMMENTED** [review]
Considering that now we have a proper `RadioGroup` component I was wondering if might be better to replace all the Confirmation with the new component?

File: src/ConfirmationRadioButtons/RadioButtonStyled.tsx (hunk 1 above)
Comment (fleonard) [reply src/ConfirmationRadioButtons/RadioButtonStyled.tsx:30]: @liampuk no what I'm saying is that the S'mores `Confirmation` button should be deprecated in favour of [this](https://marshmallow-insurance.github.io/smores-react/?path=/story/radiogroup--horizontal)

Comment (liampuk) [reply src/ConfirmationRadioButtons/RadioButtonStyled.tsx:30]: Yep that's what I mean, it's maybe worth adding a component that uses `RadioGroup` with Yes/No as that is used quite a lot in agent-portal/fnol?

Comment (fleonard) [reply src/ConfirmationRadioButtons/RadioButtonStyled.tsx:30]: you just need to pass this as options to get a YES/NO component
```
options={[
   { label: 'Yes', value: true },
   { label: 'No', value: false },
]}
 ```
 we do it already in the new direct flow in signup.

**Pull request closed without merging**